azurerm/internal/services/storage/storage_account_resource.go

```diff
@@ -314,7 +316,6 @@ func resourceStorageAccount() *pluginsdk.Resource {
 			"identity": {
 				Type:     pluginsdk.TypeList,
 				Optional: true,
-				Computed: true,
```

Comment:
how come we are removing this? and will it affect people expecting this to be populated?

Reply:
It was not `Computed`, if not set, it'll not return a default value. It causes the problem that if the user remove the block `identity`, it'll not detect the change.
But I could keep it as origin, if it breaks.